Autopilot/Path Manager/PathManager.c

```diff
@@ -667,6 +674,19 @@ void copyGPSData(){
     pmData.checkbyteDMA2 = generatePMDataDMAChecksum2();
 }
 
+static void checkForFirstGPSLock(){
+	static char gpsLockFlag = 1;
+	
+	if(((getErrorCodes() & BOR_RESET) || (getErrorCodes() & POR_RESET)) && gpsLockFlag){
```

Comment:
It'd be great if you can guide me a little here. I cant use a static variable since the value of the errors is determined at runtime and when using a non-static one, the getErrorCodes() function would be called everytime the checkForFirstGPSLock() function is executed (which is the same as now except function calls would be cut by half). The other way I can think of is using the lastRuntimeErrors variable itself from the StartUpErrorCodes.c since it is defined as a global variable there so i can extern it and use that.